aiofreepybox/api/tv.py

```diff
@@ -0,0 +1,100 @@
+class Tv:
+
+    def __init__(self, access):
+        self._access = access
+
+    async def get_finished_tv_records(self):
+        '''
+        Get finished tv records
+        '''
+        return await self._access.get('pvr/finished/')
+
+    async def get_tv_bouquet_channels(self, bouquet_id):
+        '''
+        Get tv bouquet channels
+        '''
+        return await self._access.get('tv/bouquets/{0}/channels/'.format(bouquet_id))
+
+    async def get_tv_bouquet(self):
+        '''
+        Get tv bouquet 
+        '''
+        return await self._access.get('tv/bouquets/')
+
+    async def get_tv_channels(self):
+        '''
+        Get tv channels
+        '''
+        return await self._access.get('tv/channels/')
+
+    async def get_tv_default_bouquet_channels(self):
+        '''
+        Get tv default bouquet channels
+        '''
+        return await self._access.get('tv/bouquets/freeboxtv/channels/')
```

Comment:
Are we forced to use this ? Maybe we could use a default value for `get_tv_bouquet_channels` ?

Reply:
Added default bouquet_id